### ISSUE TITLE
FR: now added data for released people 

### ISSUE THREAD

**Issue by viktoria023** [openZH/covid_19#501]
FR published numbers today for people released from hospital from the 03/04 on

**Comment by baryluk** [comment]
@viktoria023 Would be nice to provide URL to the data and link to the csv file, to make things easier for everybody.

URL is https://www.fr.ch/covid19/sante/covid-19/coronavirus-statistiques-evolution-de-la-situation-dans-le-canton

scraper does scrape this data, but it doesn't backfill it, and we only started scraping Released/Recovered around 2020-04-09, so yes, it needs to be backfilled:


FYI. Scrapes:

```
FR - - - FAILED 2020-03-27T14:05:07+01:00
FR - - - FAILED 2020-03-27T15:05:07+01:00
FR 2020-03-27T          369      15 OK 2020-03-27T16:05:09+01:00 https://www.fr.ch/covid19/sante/covid-19/coronavirus-statistiques-evolution-de-la-situation-dans-le-canton
FR 2020-03-28T          421      15 OK 2020-03-28T12:05:30+01:00 # Extras: ncumul_hosp=67 # URLs: https://www.fr.ch/covid19/sante/covid-19/coronavirus-statistiques-evolution-de-la-situation-dans-le-canton
FR 2020-03-29T          442      16 OK 2020-03-29T12:05:46+02:00 # Extras: ncumul_hosp=76 # URLs: https://www.fr.ch/covid19/sante/covid-19/coronavirus-statistiques-evolution-de-la-situation-dans-le-canton
FR 2020-03-30T          477      17 OK 2020-03-30T12:05:18+02:00 # Extras: ncumul_hosp=77 # URLs: https://www.fr.ch/covid19/sante/covid-19/coronavirus-statistiques-evolution-de-la-situation-dans-le-canton
FR -                      -       - FAILED 2020-03-30T14:05:15+02:00  # URLs: https://www.fr.ch/covid19/sante/covid-19/coronavirus-statistiques-evolution-de-la-situation-dans-le-canton # Errors: Missing date. Missing cases
FR 2020-03-30T          477      17 OK 2020-03-30T20:05:17+02:00 # Extras: ncumul_hosp=77,ncumul_ICU=16 # URLs: https://www.fr.ch/covid19/sante/covid-19/coronavirus-statistiques-evolution-de-la-situation-dans-le-canton
FR 2020-03-31T          491      20 OK 2020-03-31T15:05:17+02:00 # Extras: ncumul_hosp=77,ncumul_ICU=20 # URLs: https://www.fr.ch/covid19/sante/covid-19/coronavirus-statistiques-evolution-de-la-situation-dans-le-canton
FR 2020-04-01T          525      23 OK 2020-04-01T15:05:19+02:00 # Extras: ncumul_hosp=81,ncumul_ICU=23 # URLs: https://www.fr.ch/covid19/sante/covid-19/coronavirus-statistiques-evolution-de-la-situation-dans-le-canton
FR 2020-04-02T          550      26 OK 2020-04-02T12:05:18+02:00 # Extras: ncumul_hosp=80,ncumul_ICU=23 # URLs: https://www.fr.ch/covid19/sante/covid-19/coronavirus-statistiques-evolution-de-la-situation-dans-le-canton
FR -                      -       - FAILED -  # URLs: https://www.fr.ch/covid19/sante/covid-19/coronavirus-statistiques-evolution-de-la-situation-dans-le-canton # Errors: Missing date. Missing cases
FR 2020-04-02T          550      26 OK 2020-04-03T02:05:19+02:00 # Extras: ncumul_hosp=80,ncumul_ICU=23 # URLs: https://www.fr.ch/covid19/sante/covid-19/coronavirus-statistiques-evolution-de-la-situation-dans-le-canton
FR 2020-04-03T          588      31 OK 2020-04-03T11:42:38+02:00 # Extras: ncumul_hosp=77,ncumul_ICU=21 # URLs: https://www.fr.ch/covid19/sante/covid-19/coronavirus-statistiques-evolution-de-la-situation-dans-le-canton
FR 2020-04-04T          638      37 OK 2020-04-04T12:05:19+02:00 # Extras: ncumul_hosp=89,ncumul_ICU=21 # URLs: https://www.fr.ch/covid19/sante/covid-19/coronavirus-statistiques-evolution-de-la-situation-dans-le-canton
FR 2020-04-05T          669      40 OK 2020-04-05T15:05:37+02:00 # Extras: ncumul_hosp=87,ncumul_ICU=20 # URLs: https://www.fr.ch/covid19/sante/covid-19/coronavirus-statistiques-evolution-de-la-situation-dans-le-canton
FR 2020-04-06T          689      41 OK 2020-04-07T08:05:17+02:00 # Extras: ncumul_hosp=87,ncumul_ICU=19 # URLs: https://www.fr.ch/covid19/sante/covid-19/coronavirus-statistiques-evolution-de-la-situation-dans-le-canton
FR 2020-04-07T          729      44 OK 2020-04-07T12:05:21+02:00 # Extras: ncumul_hosp=78,ncumul_ICU=22 # URLs: https://www.fr.ch/covid19/sante/covid-19/coronavirus-statistiques-evolution-de-la-situation-dans-le-canton
FR 2020-04-08T          756      45 OK 2020-04-08T12:05:23.083045+02:00 # Extras: ncumul_hosp=80,ncumul_ICU=22 # URLs: https://www.fr.ch/covid19/sante/covid-19/coronavirus-statistiques-evolution-de-la-situation-dans-le-canton
FR 2020-04-09T          786      46 OK 2020-04-09T12:05:25.274035+02:00 # Extras: ncumul_hosp=80,ncumul_ICU=20 # URLs: https://www.fr.ch/covid19/sante/covid-19/coronavirus-statistiques-evolution-de-la-situation-dans-le-canton
FR -                      -       - FAILED 2020-04-09T13:05:22.471244+02:00  # URLs: https://www.fr.ch/covid19/sante/covid-19/coronavirus-statistiques-evolution-de-la-situation-dans-le-canton # Errors: Missing date. Missing cases
FR 2020-04-09T          786      46 OK 2020-04-09T15:05:25.445129+02:00 # Extras: ncumul_released=84,ncumul_hosp=80,ncumul_ICU=20 # URLs: https://www.fr.ch/covid19/sante/covid-19/coronavirus-statistiques-evolution-de-la-situation-dans-le-canton
FR 2020-04-10T          796      49 OK 2020-04-10T15:05:29.291048+02:00 # Extras: ncumul_released=88,ncumul_hosp=73,ncumul_ICU=19 # URLs: https://www.fr.ch/covid19/sante/covid-19/coronavirus-statistiques-evolution-de-la-situation-dans-le-canton
FR 2020-04-11T          834      53 OK 2020-04-11T12:05:31.004715+02:00 # Extras: ncumul_released=93,ncumul_hosp=68,ncumul_ICU=20 # URLs: https://www.fr.ch/covid19/sante/covid-19/coronavirus-statistiques-evolution-de-la-situation-dans-le-canton
```

**Comment by baryluk** [comment]
@viktoria023 You want to prepare a PR to fix the existing data? Or want somebody else to pick this up?

**Comment by viktoria023** [comment]
I can fill in the missing numbers. They are only available for a few days back anyway.

On 12 Apr 2020, at 08:56, Witold Baryluk <notifications@github.com<mailto:notifications@github.com>> wrote:



@viktoria023<https://github.com/viktoria023> You want to prepare a PR to fix the existing data? Or want somebody else to pick this up?

**Comment by metaodi** [comment]
Fixed with #600
